### PR TITLE
Add custom storybook theme

### PR DESCRIPTION
**Pull request by chriswynn**

### CODE REVIEW

File: .storybook/config.tsx [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+  appBg: "#1f2629",
+  appContentBg: "#171c1f",
+  appBorderRadius: 4,
+  fontBase:
+    '-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"',
+  fontCode: "monospace",
+  textColor: "white",
+  barTextColor: "white",
+  barBg: "#1f2629",
+  brandTitle: "MojoUI",
```
Comment (jssee) [inline .storybook/config.tsx]:
```suggestion
  brandTitle: "MojoTech UI",
```

File: .storybook/config.tsx [hunk 2]
```diff
@@ -1,5 +1,22 @@
-import { configure, addDecorator } from "@storybook/react";
+import { configure, addDecorator, addParameters } from "@storybook/react";
 import themeDecorator from "./theme-decorator";
+import { create } from "@storybook/theming";
+
+const theme = create({
+  base: "dark",
+  appBg: "#1f2629",
```
Comment (jssee) [inline .storybook/config.tsx]: Can we bring these hex in from `/lib/theme.js` ?

Comment (chriswynn) [reply .storybook/config.tsx]: Yeah we could but we don't have a full theme file yet. I could change that in another PR or add the theme to this one

Comment (jssee) [reply .storybook/config.tsx]: I think itd make sense to add those colors to the theme now. In general we can go adding to the theme as we use the values in the components etc. it doesnt always have to be "theme first"

**Pull request closed without merging**